lib/tests/test_wallet.py

```diff
@@ -24,6 +27,7 @@ class WalletTestCase(unittest.TestCase):
     def setUp(self):
         super(WalletTestCase, self).setUp()
         self.user_dir = tempfile.mkdtemp()
+        self.config = SimpleConfig({'electrum_path': self.user_dir})
```

Comment:
This should be, I believe, `electron_cash_path` on Electron Cash (grep the sources for this string).

Reply:
Fixed